core/src/main/java/org/bitcoinj/core/Block.java

```diff
@@ -823,6 +807,7 @@ public Sha256Hash getPrevBlockHash() {
         return prevBlockHash;
     }
 
+    @VisibleForTesting
```

Comment:
I think the addition of these annotations is also a good candidate for a separate commit.

Reply:
Agreed

Reply:
See https://github.com/bitcoinj/bitcoinj/pull/2157